libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/internal/MapboxTripNotification.kt

```diff
@@ -110,7 +105,7 @@ class MapboxTripNotification constructor(
 
     /**
      * Provides a custom [Notification] to launch
-     * with the [TripSession], specifically
+     * with the trip session, specifically
```

Comment:
I already tried. To import `TripSession` I would have to add `libnavigation-core` as a dependency to `libtrip-notification` because `TripSession` is an internal interface inside `libnavigation-core`. I cannot do this because `libnavigation-core` already depends on `libtrip-notification` and hence adding. It's better to not have [TripSession] with a broken link.

Reply:
Yeah, makes sense. Thought that `TripSession` was `public`.